src/run

```diff
@@ -19,8 +19,8 @@ base_dir=$(cd "`dirname "$loc"`" && pwd)
 # Determine os version and choose correspondent .sh or .bat file accordingly.
 os_version="`uname -a`"
 
-if [[ $os_version == *"MINGW"* ]]; then
-  exec "$base_dir/target/start.bat" "$@"
-else
+if [[ $os_version == *"MINGW"* ]]; then  
+  cmd //C "$base_dir/target/start.bat" "$@"
```

Comment:
I just reverted these files to what they are in master... I just didn't want your PR to include both CX stuff and the changes necessary to get this thing to build in Windows (which would be a totally different, and maybe pointless, effort).  So this file matches exactly what is in master